official/resnet/resnet_run_loop.py

```diff
@@ -122,9 +135,18 @@ def get_synth_input_fn(height, width, num_channels, num_classes):
     An input_fn that can be used in place of a real one to return a dataset
     that can be used for iteration.
   """
-  def input_fn(is_training, data_dir, batch_size, *args):  # pylint: disable=unused-argument
-    images = tf.zeros((batch_size, height, width, num_channels), tf.float32)
-    labels = tf.zeros((batch_size, num_classes), tf.int32)
+  def input_fn(is_training, data_dir, batch_size,  # pylint: disable=unused-argument, missing-docstring
+               use_distribution_strategy=False,
+               gpus_for_distribution_strategy=1, *args, **kwargs):  # pylint: disable=unused-argument
+    # TODO(taylorrobie@) cull DistributionStrategies uses global batch size
```

Comment:
Ditto on TODOs.

Reply:
Also, ditto on I think we can move this higher up, do once, and just have this function be blind to where the batch_size is coming from.